### PR TITLE
Add Finnish language support in GUI

### PR DESCRIPTION
**Pull request by matsduf**
This PR adds:

* `fi.json` with translation strings to Finnish
* `gui-faq-fi.md` that provides the FAQ in Finnish
* updates in configuration files to add support Finnish ("fi")
* consistent order between language codes in the configuration files
* correction of "Danish" to "Dansk" in the configuration

This PR assumes that https://github.com/zonemaster/zonemaster-engine/issues/879 and https://github.com/zonemaster/zonemaster-backend/issues/711 are resolved.

### CODE REVIEW

**Review by mattias-p — DISMISSED** [review]

File: src/assets/i18n/fi.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	"Results from previous tests are not available": "Ei aiempia testituloksia",
+	"Run test": "Suorita testi",
+	"Settings": "Asetukset",
+	"Share": "Jaa",
+	"Sorry, there is nothing here!": "Ikävä kyllä täällä ei ole mitään!",
+	"Test #": "Testinumero ",
+	"There is no delegation for the zone": "Verkkotunnuksessa ei ole delegointia",
+	"What is an undelegated domain test?": "Mikä on delegoimaton verkkotunnustesti? ",
+	"Zonemaster Backend is not available": "Zonemaster Backend ei ole käytettävissä"
+  ,"Profile"                                : "Profiili"
```
Comment (pnax) [inline src/assets/i18n/fi.json:50]: The alignment looks odd. But looking at the other translated files, they all have the same appearance for this 2 lines.

**Review by pnax — COMMENTED** [review]
This looks good. Just a comment on a wild comma being on the following line.

File: src/assets/i18n/fi.json (hunk 1 above)
Comment (samisalmensuo) [reply src/assets/i18n/fi.json:50]: This is identical with previous parts:

"Zonemaster Backend is not available": "Zonemaster Backend ei ole käytettävissä",
"Profile" : "Profiili",

Comment (matsduf) [reply src/assets/i18n/fi.json:50]: @pnax, all the XX.json look that way. I do not know why, but it does not seem to matter.

Comment (pnax) [reply src/assets/i18n/fi.json:50]: Yes I noticed this as well. I opened #186 to fix this.

Comment (matsduf) [reply src/assets/i18n/fi.json]: Fixed in new commit.

**Review by pnax — APPROVED** [review]
Beside my question on the link, this PR looks fine to me.

File: docs/FAQ/gui-faq-fi.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+1. "Engine" – testimoottori, jolla suoritetaan kaikki DNS-testit.
+2. "CLI" – testimoottorin komentorajapinta.
+3. "Backend" – palvelu, jonka avulla voit tehdä Zonemaster-testejä ja tallentaa tulokset JSON-RPC API:lla ja tietokannalla.
+4. "Backendin" ja testimoottorin verkkorajapinta.
+
+Kun verkkotunnus lähetetään Zonemasteriin, ohjelma tutkii verkkotunnuksen kunnon käymällä DNS:n läpi juuresta (.) ylätason verkkotunnukseen (esimerkiksi .NET) ja lopuksi DNS-palvelimiin, joissa on toisen asteen verkkotunnuksia (esimerkiksi zonemaster.net) koskeva informaatio. Zonemaster suorittaa myös muita testejä. Ne on kaikki dokumentoitu täällä: [Test Requirements document](https://github.com/zonemaster/zonemaster/blob/master/docs/requirements/TestRequirements.md)
+
+#### 2. Kuka on Zonemasterin takana? <a name="q2"></a>
+
+Zonemaster on yhteistyöprojekti, jonka takana ovat [Internetstiftelsen](https://internetstiftelsen.se/) (ylätason verkkotunnusten .se ja .nu hallinnoija) ja [AFNIC](https://www.afnic.fr/en/) (ylätason verkkotunnuksen .fr sekä muiden ylätason verkkotunnusten kuten .re, .pm, .tf, .wf, .yt ja .paris hallinnoija).
```
[the remark this answers is not in the document]
Comment (matsduf) [reply docs/FAQ/gui-faq-fi.md:34]: @samisalmensuo, could you comment the question?

**Review by mattias-p — APPROVED** [review]

File: docs/FAQ/gui-faq-fi.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (samisalmensuo) [reply docs/FAQ/gui-faq-fi.md:34]: This is OK to be the way it is now

[the remark this answers is not in the document]
Comment (matsduf) [reply docs/FAQ/gui-faq-fi.md:34]: Thanks!

**Pull request merged**